bin/seagull.js

```diff
@@ -2,15 +2,17 @@
 
 // core
 var path = require('path')
+var util = require('util')
 
 // npm
 var async = require('async')
+var fm = require('front-matter')
 var fs = require('graceful-fs')
+var fsExtra = require('fs-extra')
+var jade = require('jade')
+var marked = require('marked')
```

Comment:
Not sure about marked. It hasn't been updated in a while and has an outstanding edge case vuln on nodesecurity.io. But I haven't played with MD parsing lately to know if there is a better one with support for Markdown and Commonmark and GH extensions.